test/integration/util/util.go

```diff
@@ -120,7 +122,7 @@ type OcRunner struct {
 }
 
 func NewOcRunner() *OcRunner {
-	p, _ := exec.LookPath("oc")
```

Comment:
This was a crutch until we would be able to tell which oc version to use. We can now.